compiler/optimizer/OSRDefAnalysis.cpp

```diff
@@ -910,6 +912,11 @@ int32_t TR_OSRLiveRangeAnalysis::perform()
    return 0;
    }
 
+// 
+// To reduce the impact of pending push temps in postExecutionOSR, the live pending push symrefs can be stored or
+// referenced in anchored loads, between the OSR point and the transition. In the event that the anchored loads
+// are not commoned, they will be removed.
+//
```

Comment:
Okay so these loads with ref count == 1 have served their purpose when you are removing them. Thanks.